go-controller/pkg/kubevirt/pod.go

```diff
@@ -343,3 +342,10 @@ func ZoneContainsPodSubnetOrUntracked(watchFactory *factory.WatchFactory, lsMana
 	// to a node
 	return hostSubnets, !util.IsContainedInAnyCIDR(annotation.IPs[0], hostSubnets...), nil
 }
+
+// ExtractVMNameFromPod retunes namespace and name of vm backed up but the pod
+// for regular pods return nil
+func VirtualMachineOwnsPod(pod *corev1.Pod) bool {
```

Comment:
I think we should prefix the fn name with `is` or `does`, to hint the fn returns a bool.

Reply:
Renaming to `IsPodOwnByVirtualMachine`